### DISCUSSION TITLE
[General] Update in browser addon stores?

### DISCUSSION THREAD

**Discussion by dollarSignUsername** [tprouvot/Salesforce-Inspector-reloaded#554] [General]
Hi, is there a chance youd load the latest version to Firefox and (especially) Edge addon stores? Only Chrome seems to be up to date and unfortunately my user base is forced into MS ecosystem.

**Comment by tprouvot** [comment]
Hi @dollarSignUsername 

Edge is currently reviewing the 1.24 version and firefox is still in Todo

**Reply by dollarSignUsername** [reply]
Great to hear that, thank you :)
